codegen/build-oas.sh

```diff
@@ -13,7 +13,7 @@ if [ "$is_early_access" = "true" ]; then
 	template_dir="codegen/python-oas-templates/templates5.2.0"
 else
 	destination="pinecone/core/openapi"
-	modules=("control" "data")
+	modules=("db_control" "db_data")
```

Comment:
Is inference going to be a part of the plugin?

Reply:
For now, inference stuff comes in via plugin which is why you don't see anything about it here.